midp/background.js

```diff
@@ -3,20 +3,30 @@
 
 'use strict';
 
+var fgMidletNumber;
+var fgMidletClass;
```

Comment:
As a followup, we should hide these variables behind a closure (or perhaps the followup should be more general, as there are a variety of libs/* and midp/* scripts that should employ the [module pattern](http://www.adequatelygood.com/JavaScript-Module-Pattern-In-Depth.html)).